projects/valtimo/config/assets/core/nl.json

```diff
@@ -555,13 +555,17 @@
     "save": "Opslaan",
     "more": "Meer",
     "export": "Exporteren",
-    "table": {
+    "list": {
       "itemsPerPage": "Artikelen per pagina",
       "multipleSelect": "{{count}} artikelen geselecteerd",
       "ofLastPage": "van {{last}} pagina",
       "ofLastPages": "van {{last}} pagina's",
       "singleSelect": "1 artikel geselecteerd",
-      "totalItems": "{{start}}-{{end}} van {{total}} artikelen"
```

Comment:
Artikelen does not make sense to me. @peetritense could you decide for en.json, nl.json and de.json what the translation should be exactly? In the 'old list' we used the term 'results/resultaten' which I think is better. Maybe just add them directly to this PR. Also some thing are not translated right now (viewMode.table should be Tabel in Dutch etc.)